.golangci.toml

```diff
@@ -0,0 +1,8 @@
+# .golangci.toml
```

Comment:
nit: Any reason you chose the `toml` format for the config, or can we use the `.yaml` format of the config?

Reply:
No reason.  Fixed.

Also found there's a standard preset for this exclusion, so I switched to using that.